.changesets/feat_subscription_websocket_heartbeat.md

```diff
@@ -0,0 +1,16 @@
+### Subscriptions: Add configurable "heartbeat" to subgraph WebSocket protocol ([Issue #4621](https://github.com/apollographql/router/issues/4621))
+
+To account for GraphQL Subscription WebSocket implementations (e.g., [DGS](https://netflix.github.io/dgs/)) which drop idle connections by design, the router adds the ability to configure a heartbeat to keep active connections alive.
+
+An example of configuration:
+
+```yaml
+subscription:
+  mode:
+    passthrough:
+      all:
+        path: /graphql
+        heartbeat_interval: enable #Optional
```

Comment:
Why are we introducing `enable`?
The default before was 5s.

Reply:
I was under the impression that it was disabled by default before?

Reply:
Yes, it's disabled by default.
Benjamin suggested to add `enable` since we already have 5s interval as default for callback protocol.
The idea was to allow customers to set reasonable defaults without trying to guess the numbers.

Reply:
Enable is equal to 5s, which is our current default for callback protocol.